app/assets/javascripts/discourse/app/components/sidebar/common/community-section.js

```diff
@@ -35,7 +35,16 @@ export default class SidebarCommunitySection extends Component {
   }
 
   willDestroy() {
-    this.sectionLinks.forEach((sectionLink) => sectionLink.teardown());
+    [
+      ...this.sectionLinks,
+      ...this.moreSectionLinks,
+      ...this.moreSecondarySectionLinks,
+    ].forEach((sectionLink) => {
+      if (sectionLink.teardown) {
+        sectionLink.teardown();
```

Comment:
minor not: I know in chat we've been doing this form:

```javascript
this.onChangeSelectedUsers?.(this.selectedUsers);
```

So maybe we could do that here for consistency?

```javascript
sectionLink.teardown?.()
```

Reply:
O this is nicer :) I've updated the code